src/_scss/pages/agencyV2/statusOfFunds/_introSection.scss

```diff
@@ -0,0 +1,34 @@
+.status-of-funds__intro-wrapper {
+    @include span-columns(16);
+    text-align: left;
+    margin: auto;
+    padding-bottom: 4rem;
+    @media (min-width: $medium-screen) {
+        padding-left: 4.8rem;
+    }
+}
+
+.status-of-funds__intro-section-title {
+    font-size: 2rem;
+    font-weight: 600;
+    line-height: 2rem;
+    margin-bottom: 2rem;
+}
+
+.status-of-funds__intro-section-text {
+    font-size: 1.4rem;
+    font-weight: 400;
+    line-height: 2rem;
+    margin-bottom: 2rem;
+}
+
+.status-of-funds__glossary-term {
+    font-weight: $font-semibold;
```

Comment:
One minor thing I noticed - in the mockups the glossary links are gray instead of blue. If you import `components/glossaryLink` here that should fix it!

Reply:
Good eye, thank you.

Reply:
Sorry, I see that the icon is not actually nested inside the class `status-of-funds__glossary-term`, so this import should be moved outside this block to work.